data/mods/National_Guard_Camp/military.json

```diff
@@ -2,8 +2,8 @@
   {
     "id": "mon_w11b10",
     "type": "MONSTER",
-    "name": "W11B10",
-    "description": "Wraitheon (11B) Infantry 10 Level.  Part of Wraitheon's series of one-to-one auxiliaries designed to seamlessly integrate with more traditional forces.",
+    "name": "rifle bot",
```

Comment:
Why not "W11B10 rifle bot"? "rifle bot" is way too generic.

Reply:
Sure, I'll change it.